_docs/2.0.0/usage/schema.md

```diff
@@ -252,18 +260,19 @@ The type is specified as struct of following properties:
+| [default](#default)                       | any atomic type             | Default value to use in case data are missing                                                                                                         | *0*             | `-`[\*\*](#metadata-star-star)           |
+| [pattern](#pattern)                       | timestamp & date            | Pattern for the timestamp or date representation                                                                                                      | *dd.MM.yy*      | *yyyy-MM-dd HH:mm:ss* **/** *yyyy-MM-dd* |
+| [timezone](#timezone)                     | timestamp (also date)       | The time zone of the timestamp when that is not part of the pattern (NB! for date it can return unexpected results)                                   | *US/Pacific*    | *UTC*[\*\*\*](#metadata-star-star-star)  |
+| [pattern](#pattern)                       | any numeric type            | Pattern for the number representation                                                                                                                 | \#,\#\#0.\#     | `-`[\*\*](#metadata-star-star)           |
+| [decimal_separator](#decimal_separator)   | any numeric type            | The character separating the integer and the fractional parts of the number                                                                           | *,*             | *.*                                      |
+| [grouping_separator](#grouping_separator) | any numeric type            | Character to mark boundaries between orders of magnitude, usually to mark thousands, millions etc.                                                    | *\_*            | *,*                                      |
+| [minus_sign](#minus_sign)                 | any numeric type            | Character to mark the number is negative.                                                                                                             | *N*             | *-*                                      |
+| [allow_infinity](#allow_infinity)         | float & double              | Flag indicating if the column accepts infinity as a value (and positive/negative numbers which are too large are converted to *infinity*/*-infinity*) | *true*          | *false*                                  |
+| [radix](#radix)                           | long, integer, short, byte  | The base of the numbers provided                                                                                                                      | *hex*           | *10*                                     |
+| [encoding](#encoding)                     | string ("base64" or "none") | Encoding used for string->binary conversion                                                                                                           | *base64*,*none* | (no encoding present)                    |
```

Comment:
I would say that the correct line would be this.
- It is simply on a binary. Description says more
- It is for a string to binary conversions.
- And the default is none (or not used), but not used does not have to be there. Then we would have to say it everywhere. That doesn't make sense

```suggestion
| [encoding](#encoding)                     | binary                      | Encoding is used for string to binary conversion                                                                                                      | *base64*,*none* | *none*                                   |
```

Reply:
ok